data_retriever/migration_event.py

```
@@ -1,6 +1,9 @@
 from dataclasses import dataclass
+from enum import Enum
 from json import dumps as json_dumps, loads as json_loads
```

Ruff (0.12.2)

`json.loads` imported but unused

Remove unused import: `json.loads`

(F401)